src/compiler/rust/canister/src/ic/notify_raw.rs

```diff
@@ -46,7 +46,7 @@ impl JsFn for NativeFunction {
 
                 // Err(anyhow::anyhow!(err_string))
 
-                panic!(err_string);
+                panic!("{}", err_string);
```

Comment:
How was this not broken before?

Reply:
it might not have been a breaking problem, I saw it while I was going through rust errors from verbose, it was only listed as a warning